### PR TITLE
Nuke IOWorker, make stuff compile

### PR DESCRIPTION
**Pull request by MiniDigger**

### CODE REVIEW

**Comment by MiniDigger** [comment]
ty travis for the green checkmark <3

**Comment by MiniDigger** [comment]
thank you @Spottedleaf for assisting in getting chunk stuff right

**Comment by MiniDigger** [comment]
:shipit: 
shipit

File: Spigot-Server-Patches/0394-Asynchronous-chunk-IO-and-loading.patch [hunk 1]
```diff
@@ -1,10 +1,11 @@
-From 867219c4f28a3b9b5ea9dfe010569d45b620f1e1 Mon Sep 17 00:00:00 2001
+From 2569817269d8854ce427a42e1880785f5ab7cf26 Mon Sep 17 00:00:00 2001
 From: Spottedleaf <Spottedleaf@users.noreply.github.com>
 Date: Sat, 13 Jul 2019 09:23:10 -0700
 Subject: [PATCH] Asynchronous chunk IO and loading
 
 THIS PATCH NEEDS RE-EVALUTING AND WILL LIKELY NOT WORK AS-IS RIGHT THIS SECOND
```
Comment (electronicboy) [inline Spigot-Server-Patches/0394-Asynchronous-chunk-IO-and-loading.patch]: Feel free to remove this warning, this is basically what I was planning to do, our IO stuff is nicer

**Comment by MiniDigger** [comment]
https://github.com/MiniDigger/Paper/tree/wip/1.15/removed/1.15
I feeel like we can easily port a bunch of these 
I am heading out now tho so feel free to take over

**Review by electronicboy — APPROVED** [review]

**Pull request merged**